docs/fundamentals/extensibility/extending-domain-models.md

```diff
@@ -195,60 +174,3 @@ There is a technical task of an instantiation of the right “effective” type
 There are two ways to force ASP.NET Core API Json serializer to understand our domain extensions:
 1. Use platform-defined `PolymorphJsonConverter`. It's preferable to use in most cases. The `PolymorphJsonConverter` transparently deserializes extended domain types with no developer effort.
 2. Custom JSON converter passed to MvcNewtonsoftJsonOptions. Consider using it, if `PolymorphJsonConverter` is not suitable for your specific case.
-
-### Universal `PolymorphJsonConverter`
```

Comment:
Do we really wanna remove all this stuff?

Reply:
Yes. @tatarincev said it should be in black box